lua/nvim-lsp-installer/servers/omnisharp/init.lua

```diff
@@ -7,55 +7,83 @@ local std = require "nvim-lsp-installer.core.managers.std"
 
 local coalesce, when = functional.coalesce, functional.when
 
+function GenerateCmd(mono_path, root_dir)
+  if mono_path == nil and vim.fn.executable("mono") then
+    mono_path = "mono"
+  end
```

Comment:
I don't see a reason to take a `mono_path` as a setting? Wouldn't it be enough to assume it is on path, and instead let the user decide via a boolean flag, like

```lua
lspconfig.omnisharp.setup {
  use_mono = true
}
```

Or am I missing something here?

Reply:
If mono is available I think it is preferred to use the mono version, this is because the dotnet version as far as I understand is a minimal version that is not supporting external libraries, other than that the two are identical as far as I know and the dart version is self sufficient to run.

All that said, if mono is available, we should use it, if it is not available we use the dotnet version. Since other servers are not taking any binary path we can drop `mono_path` altogether and not introduce `use_mono` neither if it sounds good to you.